app/models/comment.php

```diff
@@ -0,0 +1,46 @@
+<?php
+class Comment extends AppModel
+{
+	public $validation = array(
+		'body' => array(
+			'length' => array(
+				'validate_between', 1, 200,
+				),
+			),
+		);
+
+	public function getAll($offset, $limit)
+	{
+		$comments = array();
+		$thread = Param::get('thread_id');
+		$db = DB::conn();
+
+		$rows = $db->rows(
+		"SELECT * FROM comment WHERE thread_id = ? ORDER BY created ASC LIMIT {$offset},{$limit}", array($thread));
+
```

Comment:
You may remove ```ASC``` since the ```ORDER BY``` keyword in MySQL sorts the records in ascending order by default. :bow: